lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -3218,8 +3218,12 @@ class MMA16816SmemLoader {
     cMatShape = matShape[order[0]];
     sMatShape = matShape[order[1]];
 
-    cTileStride = smemStrides[order[0]];
-    sTileStride = smemStrides[order[1]];
+    // TODO[Superjomn]: Seems the smemStrides not works here(the master branch
+    // will fail too).
```

Comment:
Is it possible for us to catch this kind error with filecheck test or unit test?

Reply:
Yeah, it is caught in a test case.

Reply:
Seems good to me. Glad that we can pass all tests.

Reply:
Yeah, the misaligned error must because of the 1 from `16x1`.